lib/langserver.js

```diff
@@ -0,0 +1,58 @@
+const { URI } = require('vscode-uri');
+const { promises: fs } = require('fs');
+const { hasOctaneImports, transformOctaneImports } = require('./octane-utils');
+const { transformImports } = require('./utils');
+
+async function onComplete(projectRoot, opts) {
+  function log() {
+    return opts.server.connection.console.log(...arguments);
+  }
+
+  if (opts.focusPath.node.type === 'ElementNode') {
+    let filePath = uriToPath(opts.textDocument.uri);
```

Comment:
const documentContent = opts.textDocument.getText();

Reply:
hum, `textDocument` only has `uri` on it.